lib/command/src/Obelisk/Command/Thunk.hs

```diff
@@ -349,17 +371,32 @@ parseGitSource v = do
     , _gitSource_private = fromMaybe False private
     }
 
-overwriteThunk :: MonadObelisk m => FilePath -> ThunkPtr -> m ()
-overwriteThunk target thunk = do
+overwriteThunk :: MonadObelisk m => Bool -> FilePath -> ThunkPtr -> m ()
+overwriteThunk preserveSymlinks target thunk = do
   -- Ensure that this directory is a valid thunk (i.e. so we aren't losing any data)
   readThunk target >>= \case
     Left e -> failWith [i|Invalid thunk at ${target}: ${e}|]
     Right _ -> pure ()
 
+  unsafeOverwriteThunk preserveSymlinks target thunk
+
+unsafeOverwriteThunk :: MonadObelisk m => Bool -> FilePath -> ThunkPtr -> m ()
+unsafeOverwriteThunk preserveSymlinks target thunk = do
   --TODO: Is there a safer way to do this overwriting?
-  liftIO $ removePathForcibly target
+  removeThunkPath preserveSymlinks target
   createThunk target $ Right thunk
 
+removeThunkPath :: MonadIO m => Bool -> FilePath -> m ()
+removeThunkPath preserveSymlinks target = liftIO $ do
+  if preserveSymlinks
+    then do
+      ps <- listDirectory target
+      forM_ ps $ \p -> do
+        removePathForcibly (target </> p)
+    else do
+      putStrLn ("removing " ++ show target)
```

Comment:
We need to use obelisk's logging facility here and capitalize log messages.

Reply:
oops, that's a wayward debugging statement.  It can be removed altogether as far as I'm concerned.